src/main/scala/com/lightbend/lagom/core/persistence/effects/core/persistence/AccountEntity.scala

```diff
@@ -36,8 +49,8 @@ class AccountEntity extends PersistentEntity[AccountCommand[_], AccountEvent, Ac
       }
 
   // on creation and later
-  val depositCommandActions =
-    actions
+  val depositCommandActions: Actions =
+    actionBuilder
```

Comment:
The reason why I didn't append it with `Builder` is to make it sound less technical. 
On this other PR https://github.com/lagom/persistence-api-experiments/pull/2, I even removed the need of calling `actions`.

You can simply start with `onCommand` and `onEvent`